src/extension/background-script/events/persistPayments.js

```diff
@@ -4,18 +4,17 @@ const persistSuccessfullPayment = async (message, data) => {
   const recipientName = data.origin.name;
```

Comment:
Can this be switched to TS without bigger issues?

Reply:
good point. 
but I just tried it and it is a bit of a bigger thing because the values are currently all `any` and have to be defined. 
wanna pair on this next week?

Reply:
Yay pairing! Yes, let's!

Reply:
I merge this fix now, and we pair on TS for a new PR next week? ok?

Reply:
Yup